src/dev/flang/fe/LibraryFeature.java

```diff
@@ -868,7 +875,57 @@ else if (other instanceof LibraryFeature lf)
     return result;
   }
 
+  /**
+   * Union of the library modules of all inner features. Checks inner features recursively.
+   * @return immutable set with all library features for which an inner feature exists
```

Comment:
also here, library modules instead of library features?

Reply:
🙈